### PR TITLE
FH-429: add view version option

### PR DESCRIPTION
**Pull request by jerrieyang**

### CODE REVIEW

File: src/formhero.js [hunk 1]
```diff
@@ -339,13 +339,18 @@ var formhero = (function (api) {
             {
                 modeParm = '&mode=' + modeParam;
             }
+
+            const previewUrl = (options.versionNumber) ? [['form-preview/'],['/', options.versionNumber, '/']] : [[],[]];
+
             var formUrl = [
```
Comment (asulaiman) [inline src/formhero.js]: This form of joining isnt necessary. We should change to : 
`${hostDetails.protocol}${formheroHost}/#/${options.versionNumber ? `form-preview/` : ''}${encodeURIComponent(options.team)}/${encodeURIComponent(options.form)}/${options.versionNumber ? `${options.versionNumber}` : ''}/?new=true${modeParam}`

Comment (jerrieyang) [reply src/formhero.js]: tried... make it so hard to read...

Comment (jerrieyang) [reply src/formhero.js]: I think use join is a clever way to handle this :).... 
Isn't this too long to read?
`${hostDetails.protocol}${formheroHost}/#/${(options.cuid)?'form-preview/':''}${encodeURIComponent(options.team)}/${encodeURIComponent(options.form)}${........}?new=true${modeParam}`

Comment (asulaiman) [reply src/formhero.js]: clever but hard to figure out whats being done, took me a while to figure it out.

Comment (jerrieyang) [reply src/formhero.js]: hahaha... didnt take long for me... i would leave the way it is... if u dont mind..

Comment (asulaiman) [reply src/formhero.js]: alright

**Pull request merged**